src/Test/DockerIntegration/ConfigDumpTest.php

```diff
@@ -0,0 +1,72 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+namespace Magento\MagentoCloud\Test\DockerIntegration;
+
+use Magento\MagentoCloud\Test\DockerIntegration\Process\Copy;
+use Magento\MagentoCloud\Util\ArrayManager;
+
+/**
+ * @inheritdoc
+ *
+ * @php 7.2
+ */
+class ConfigDumpTest extends AbstractTest
+{
+    public function test()
+    {
+        (new Process\GitClone('master'))
+            ->setTimeout(null)
+            ->mustRun();
+        (new Process\ComposerInstall())
+            ->setTimeout(null)
+            ->mustRun();
+
```

Comment:
Should we look to start abstracting some of these out to helpers?  For instance, the above 2 sets of code seem to be called in every test.  I'm also thinking we could combine some of these tests for speed (though I can't quite see the design in my head right now) but for example, this test and AcceptanceTest are the exact same from lines 18-41 so maybe those could, at a minimum, be moved to helper functions, but even possibly be combined so they are a single deploy process

Reply:
Merged ConfigDumTest into AcceptanceTest